lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2093,6 +2093,7 @@ definitions:
       - ptp_connected
       - mme_connected
       - fsm_state
+      - ip_address
```

Comment:
Breaks backward compatibility fyi

Reply:
For now I'll add ip_address as not required to not break this, we mainly need this to filter enb metrics and display it on NMS.